src/lib/components/AdditionalDescriptionsField.js

```diff
@@ -16,6 +16,10 @@ import { LanguagesField } from './LanguagesField';
 import { i18next } from '../i18next';
 
 export class AdditionalDescriptionsField extends Component {
+  sortOptions = (options) => {
```

Comment:
This function is now repeated over 3 different components... can we make it generic somehow? It wouldn't make sense in `Field`... Ideas?

Reply:
Until there is a better place, we can place it here: https://github.com/inveniosoftware/react-invenio-deposit/blob/master/src/lib/utils.js 

```js
function sortOptions(options) {
    return options.sort((o1, o2) => o1.text.localeCompare(o2.text));
}
```

Reply:
Thanks!